resources/js/homeAdmin.js

```diff
@@ -0,0 +1,47 @@
+import Swal from 'sweetalert2';
+
+$(function(){
+    $('#button-logout').click(()=> {
+        $('#logout-form').submit();
+    });
+
+    $('.table.table-sortable th.sortable').click(function() {
+        var o = $(this).hasClass('sort-asc') ? 'sort-desc' : 'sort-asc';
+        $(this).parents('table').find('th.sortable').removeClass('sort-asc').removeClass('sort-desc');
+        $(this).addClass(o);
+    });
+
+    $.ajaxSetup({
+        headers: {
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        }
+    });
+
+    $('.check-toggler').on('click', function(e){
+        console.log(e.target.getAttribute('id'));
+        let abc = e.target.getAttribute('id');
+        Swal.fire({
+            title: 'Are you sure?',
+            text: "You won't be able to revert this!",
+            icon: 'warning',
+            showCancelButton: true,
+            confirmButtonColor: '#3085d6',
+            cancelButtonColor: '#d33',
+            confirmButtonText: 'Yes!'
+        }).then((result) => {
+            if (result.isConfirmed) {
+                $.ajax({
+                    type: 'put',
+                    url: `/updateStatusPost/${abc}`,
+                    success: function () {
+                        location.reload();
+                    },
```

Comment:
dùng ajax mà load lại trang thì làm gì e?

Reply:
em dùng ajax để truyền dữ liệu vào controller để update Models. Cơ mà dữ liệu đc in ra ở blade thì đc gửi từ controller nên e phải dùng cái này để load lại dữ liệu ạ

Reply:
c có thấy em truyền data gì đâu? dữ liệu gửi sang phải dùng ajax mà in ra chuws?